librarian/core/contrib/templates/decorators.py

```diff
@@ -8,7 +8,14 @@
 file that comes with the source code, or http://www.gnu.org/licenses/gpl.txt.
 """
 
-import functools
+
+class AttrDict(dict):
```

Comment:
`AttrDict` looks like a class which could be used project wide. Can this be moved to a more general module like say `librarian.core.utils`